delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/client/S3StorageClient.scala

```diff
@@ -35,27 +37,32 @@ object S3StorageClient {
         StaticCredentialsProvider.create(
           AwsBasicCredentials.create(cfg.defaultAccessKey.value, cfg.defaultSecretKey.value)
         )
-
-      Interpreter[IO]
-        .S3AsyncClientOpResource(
-          S3AsyncClient
-            .builder()
-            .credentialsProvider(creds)
-            .endpointOverride(URI.create(cfg.defaultEndpoint.toString()))
-            .forcePathStyle(true)
-            .region(Region.US_EAST_1)
-        )
-        .map(new S3StorageClientImpl(_, cfg.defaultEndpoint))
+      resource(URI.create(cfg.defaultEndpoint.toString()), creds)
```

Comment:
You can just use URI in the S3StorageConfig and get rid a akka one.
(There may be something to write a decoder/encoder for Circe but it should be easy)